src/app/jobs/components/job-list/job-list.component.html

```diff
@@ -29,6 +29,11 @@ <h1>Jobs</h1>
         <td mat-cell *matCellDef="let job">{{job.product_name}}</td>
       </ng-container>
 
+      <ng-container matColumnDef="organism">
+        <th mat-header-cell *matHeaderCellDef>Organism</th>
+        <td mat-cell *matCellDef="let job" matSortActive="name" matSortDirection="asc">{{getOrganismNameById(job.organism_id) | async}}</td>
```

Comment:
Please change the default sort, the table should be sorted from newest to oldest jobs (Started column)